### ISSUE TITLE
GAN loss for the first 200k steps

### ISSUE THREAD

**Issue by tebin** [mindslab-ai/univnet#4]
The paper says 
> We trained the generator with only auxiliary loss without discriminators in the first 200k steps.

but I don't think your training code reflects that, and starts combining `stft_loss` and `score_loss` right off the bat at step 0. Is there any reason behind this modification?

**Comment by wookladin** [comment]
Hi!

There is no particular reason for using GAN Loss from the beginning.
However, we found that the perceptual qualities of the samples and objective metrics match the results of the paper even if GAN Loss was used from the beginning for the c16 configuration.
You can refer to [README.md](https://github.com/mindslab-ai/univnet/blob/master/README.md) to find objective results of ours and the paper.

I am currently training c32, and the results seem to be probably similar to the results of the paper.

Thanks for your concern!

**Comment by enhuiz** [comment]
Not using GAN losses in the beginning stage makes the training much faster. I tried to train the model on an RTX 3060, it takes ~1.1s per iteration with the GAN losses but only ~0.25s per iteration without GAN losses.
